src/refract/parseResult.es6

```diff
@@ -0,0 +1,48 @@
+/*
+ * Parse result-specific refract elements. Includes the API namespace.
+ * General structure:
+ *
+ * + ParseResult
+ *   + Annotation
+ */
+
+import {
+  ArrayElement, StringElement, registry
+} from 'minim';
+
+import './api';
+
+export class ParseResult extends ArrayElement {
+  constructor(...args) {
+    super(...args);
+    this.element = 'parseResult';
+  }
+
+  get api() {
+    return this.findByClass('api').first();
+  }
+
+  get annotations() {
+    return this.findByElement('annotation');
+  }
+}
+
+export class Annotation extends StringElement {
+  constructor(...args) {
+    super(...args);
+    this.element = 'annotation';
+  }
+
+  get code() {
+    return this.attributes.getValue('code');
+  }
+
+  set code(value) {
+    this.attributes.set('code', value);
+  }
```

Comment:
@pksunkara it's unclear to me where and how the `sourceMap` element type is actually used. No examples include `element: "sourceMap"` anywhere. Is this something that should be supported or do we need to revisit the spec?